boards/rp-pico/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "pico"
+name = "rp-pico"
 version = "0.1.0"
```

Comment:
Version needs to be higher for `rp-pico`, as there is already an older BSP with version `0.1.3` with that name.
So bump it to `0.2.0`, I guess?

Reply:
https://github.com/rp-rs/rp-hal/pull/245/commits/6271e8670fe61cafb48f636ceaf5e001949a1442